### DISCUSSION TITLE
Events with multiple subjects involved

### DISCUSSION THREAD

**Discussion by tbortolini** [olivierfriard/BORIS#579] [Q&A] [answered]
Hi, I am coding video files with human participants performing a group task. The composition of the group changes during each session, but the codes are always the same (e.g. participants A,B,C doing behavior 1, later on participants A,C,D performs behavior 1, E,F,G performs behavior 2, etc, etc).

Currently I am creating a "new subject" for each composition and coding  the behavior performed by that group of participants. However, I am ending up with a huge list of "subjects" due to all the possible combinations of participants in a single session (A,B,C; A,C,E; A,B,F,G, and so on). 

Is there a better way to assign more than one participant to a single event? (e.g. have simple list of single participants A to G and then when I code an event, I can list the ones involved with it).

I tried to look in the manual and discussion list and couldn't find it, thank you for any suggestions! If that is not possible yet, I believe it might be an interest new feature.

All the best,

Tiago

**Comment by olivierfriard** [comment, marked as answer]
Hi,
you can add more than one participant to an event by using a modifier. You can create a **Multiple selection** modifier and load the subject list (by using the **Add subject as modifiers** button).
After each event trigger, BORIS will let you select the participants from a list

See https://boris.readthedocs.io/en/latest/#set-the-modifiers for details

Bye

Olivier

**Reply by tbortolini** [reply]
Thanks! 
Sorry, I missed the Multiple selection modifiers session in the documentation.
